schemas/plugindefinition-schema.json

```diff
@@ -228,6 +228,22 @@
         "framework": {
           "type": "string",
           "enum": [ "angular2", "angular", "react" ]
+        },
+        "entryPoint": {
+          "type": "object",
+          "description": "Describes a path, relative to the app's '/web' folder in the browser, where the main file is that will be loaded by the Zowe Desktop. This is assumed to be 'main.js' for Zowe versions older than v3 but allows you to ship multiple entryPoints to target different versions of Zowe",
+          "additionalProperties": false,
+          "properties": {
+            "2.0": {
```

Comment:
This won't be for 2.0, 3.0 won't be for 3.0 etc. this will be for V2 or V3 right? So let's change it to 
"v2" or "2" and "v3" or "3"

Reply:
its intended to be a version number, so that if a change is needed there can be a "3.1", "3.2", etc.